cmd/createcluster.go

```diff
@@ -152,22 +147,30 @@ func runCreateCluster(ctx context.Context, w io.Writer, conf clusterConfig) erro
 		if err != nil {
 			return err
 		}
+
+		conf.FeeRecipientAddrs = def.FeeRecipientAddresses()
```

Comment:
it feels like definition is the source of truth from this point, why is it required to update the config? The else statement creates a definition from the config, feels like we should just use definition from this point forward